src/components/utrecht/TableFooter.tsx

```diff
@@ -0,0 +1,22 @@
+/**
+ * @license EUPL-1.2
+ * Copyright (c) 2022 Robbert Broersma
+ */
+
+import clsx from "clsx";
+import { ForwardedRef, forwardRef, PropsWithChildren, TableHTMLAttributes } from "react";
+
+type TableFooterProps = TableHTMLAttributes<HTMLTableSectionElement>;
```

Comment:
I think this should be HTMLAttributes, or if it exists TfootHTMLAttributes instead